runtime-light/utils/php_assert.cpp

```diff
@@ -61,5 +60,5 @@ void php_error(char const *message, ...) {
 void php_assert__(const char *msg, const char *file, int line) {
   php_error("Assertion \"%s\" failed in file %s on line %d", msg, file, line);
   critical_error_handler();
-  _exit(1);
+  k2::exit(1);
```

Comment:
I don't think `k2::exit` is necessary here. It already done in `critical_error_handler`

Reply:
I think it makes it clear that we pass the control to the platform.